hs/app/reach/Main.hs

```diff
@@ -241,9 +250,11 @@ mkVar = do
   reachEx <- lookupEnv "REACH_EX"
     >>= maybe (die "Unset `REACH_EX` environment variable") packed
   connectorMode <- do
-    rcm <- m "REACH_CONNECTOR_MODE" (pure "ETH-devnet") (pure . id)
+    let p = "REACH_CONNECTOR_MODE"
+    let d = "ETH-devnet"
```

Comment:
Since we last talked I've convinced myself that no-default is a good idea. I don't think you need to do it in this PR, because I like incrementalism, but I think you should make a card and do it in this spring.